mods/xpanes/init.lua

```diff
@@ -160,6 +160,19 @@ xpanes.register_pane("pane", {
 	}
 })
 
+xpanes.register_pane("obsidian_pane", {
+	description = "Obsidian Glass Pane",
+	textures = {"default_obsidian_glass.png","xpanes_pane_half.png","default_obsidian.png"},
+	inventory_image = "default_obsidian_glass.png",
+	wield_image = "default_obsidian_glass.png",
+	sounds = default.node_sound_glass_defaults(),
+	groups = {snappy=2, cracky=3, oddly_breakable_by_hand=3},
```

Comment:
Obsidian glass, unlike normal glass, does not have the oddly_breakable_by_hand group, so should not have that here. Full support from me if that's removed.

Reply:
done